### ISSUE TITLE
Hotlinking Imgur images does not work

### ISSUE THREAD

**Issue by softyoda** [NUKnightLab/juxtapose#175]
Hello, i'm trying to make a comparaison slider for the first time

preview work fine, but published link ( [https://cdn.knightlab.com/libs/juxtapose/latest/embed/index.html?uid=d2a96230-c791-11eb-b7bf-95443c729a29](url) ) only show a blank page (tried on last version of firefox, chrome, edge on windows 10 64bit)

**Comment by JoeGermuska** [comment]
It is against Imgur's terms of service to "hotlink" images (use the "direct" image URL). While you are authoring, it may appear to work, because your browser has the images cached; however, when the images are not in the cache, the image space will appear blank, and the browser's network console will show that Imgur returned a 403 "Forbidden" error when the images were requested.

While this is not an endorsement, We've recently learned about postimages.org which seems to be a good service for hosting images. After you upload an image there, use the "Direct link" as the URL you enter in the Juxtapose configuration.

In short, this is not a Juxtapose bug, so closing as INVALID.

**Comment by softyoda** [comment]
Ok, I didn't know about that, thanks for the answer.

I wanted a quick solution and there were no other solution that juxtapose and a self-hosted site.

But I found another way with codepen : https://codepen.io/softyoda/full/OJpEgWL
